client/src/Login.tsx

```diff
@@ -1,63 +1,147 @@
-import React, { FormEvent, useState } from 'react';
-import { useNavigate } from 'react-router-dom';
+import React, { FormEvent, MouseEventHandler, useEffect, useState } from 'react';
+import { useNavigate, useLocation } from 'react-router-dom';
 import './styles/login.scss';
+import LoadingScreen, { LoadingScreenType } from './components/loadingScreen';
+import logo from '../../images/konstellation-logo.png'
+import background from './styles/images/login-background.png'
+/**
+ * Catch their respective event from the electron main process
+ * Need to be outside of the Login component, otherwise they will fire twice
+ */
 
 function Login() {
   let navigate = useNavigate();
+  // Message to send to the user on invalid input
+  const badCredentials = "Invalid Credentials"
+  // State will be null, but won't be if navigated from app
+  // Don't autoload user into app if they navigated from app
+  let { state } = useLocation();
+  const autoLoad = state === null ? true : false;
+
   const [accessKey, setAccessKey] = useState("");
   const [secretKey, setSecretKey] = useState("");
   const [clusterName, setClusterName] = useState("");
   const [regionName, setRegionName] = useState("");
+  // Toggling Password visiblity
+  const [hideEye, toggleEye] = useState(true);
+  const [hidePassword, togglePassword] = useState(true);
+  // Whether to show the loading screen
+  const [isLoading, setLoading] = useState(false);
+
+  // Only fire on initial render
+  // Executes twice because of React.Strict
+  useEffect(() => {
+    // On mount
+    // Don't load user in if they just logged out
```

Comment:
Don't think we need to call out:
- Then load config
- On Mount
- Only Fire on Initial Render
-  Whether to show loading screen
-  Toggle Password Visibility
- I wouldn't leave a comment that is a question (line 55). Simply make a note of it for other developers and reference it as a bug.
- Line 60 (update each input field) also feels unecessary.